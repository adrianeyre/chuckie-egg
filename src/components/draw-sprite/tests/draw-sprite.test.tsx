import React from 'react';
import { shallow } from 'enzyme';

import DrawSprite from '../draw-sprite';
import IDrawSpriteProps from '../interfaces/draw-sprite-props';
import Player from '../../../classes/player';

describe('Draw Sprite', () => {
	it('Should render correctly', () => {
		const defaultProps: IDrawSpriteProps = {
			sprite: new Player({}),
			image: 'image',
		};

		const drawFish = shallow(<DrawSprite {...defaultProps} />);
		expect(drawFish).toMatchSnapshot();
	});
});