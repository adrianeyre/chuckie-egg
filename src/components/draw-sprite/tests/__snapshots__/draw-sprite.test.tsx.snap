// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Draw Sprite Should render correctly 1`] = `ShallowWrapper {}`;
